src/Diagnostics/cfl.jl

```diff
@@ -85,4 +85,3 @@ DiffusiveCFL(Δt) = CFL(Δt, cell_diffusion_timescale)
 
 using Oceananigans.Advection: cell_advection_timescale
```

Comment:
```suggestion
```

Reply:
@glwagner won't we need a method for `cell_advection_timescale(model)`?

Reply:
There are a bunch of `accurate_cell_advection_timescale` calls in tests and validations scripts btw we should cleanup.

Reply:
delete them all? a great chance to delete lines.